x-pack/plugins/uptime/public/routes.tsx

```diff
@@ -151,21 +154,38 @@ export const PageRouter: FC = () => {
     }
   `;
 
+  const { data } = useSelector(indexStatusSelector);
+  const noDataInfo = !data || data?.docCount === 0 || data?.indexExists === false;
+  const noDataConfig: KibanaPageTemplateProps['noDataConfig'] = {
```

Comment:
**Uptime**

Reply:
Looks like this logic may not be quite right: https://github.com/elastic/kibana/pull/107709/checks?check_run_id=3318902060